### PR TITLE
Version 2.10.6 adding missing changelog

### PR DESCRIPTION
**Pull request by yash-tingre-10**
### Description
Added change log for Image gallery and eye icon not visible

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -216,10 +216,12 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 == Changelog ==
 
 = 2.10.6 - WEDNESDAY, 27th DECEMBER 2023 =
+* Feature: Transform to - Added transform to option for Image-Gallery.
 * Improvement: Caching - Added compatibility with Cloudway's caching - Breeze plugin.
 * Fix: FAQ - Resolved an issue wherein FAQ blocks inside an inner-container would shrink when minimized.
 * Fix: Tabs - Resolved an issue wherein the nested tab does not shows the content on the frontend.
 * Fix: Global - Resolved an issue wherein the template styles for static page selected as front-page would not load as intended in FSE themes.
+* Fix: Google Maps - Resolved an issue where the eye icon to hide the page title disappears when importing Google Maps in the editor.
```
Comment (AbhijitNage123) [inline readme.txt:224]: @yash-tingre-10  Google map changelog should be above the global changelog.

Comment (yash-tingre-10) [reply readme.txt:223]: Done

**Review by AbhijitNage123 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -216,9 +216,11 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 ## Changelog ##
 
 ### 2.10.6 - WEDNESDAY, 27th DECEMBER 2023 ###
+* Feature: Transform to - Added transform to option for Image-Gallery.
```
Comment (AbhijitNage123) [inline README.md:219]: Transform from Core Gallery to Spectra Image Gallery.

**Comment by AbhijitNage123** [comment]
@yash-tingre-10 Fix: Slider - Resolved an slides per view ( pro option ) not working in editor.

**Review by AbhijitNage123 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -216,9 +216,12 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 ## Changelog ##
 
 ### 2.10.6 - WEDNESDAY, 27th DECEMBER 2023 ###
+* Feature: Transform to - Added transform to option for core gallery to spectra Image Gallery.
 * Improvement: Caching - Added compatibility with Cloudway's caching - Breeze plugin.
 * Fix: FAQ - Resolved an issue wherein FAQ blocks inside an inner-container would shrink when minimized.
 * Fix: Tabs - Resolved an issue wherein the nested tab does not shows the content on the frontend.
+* Fix: Google Maps - Resolved an issue where the eye icon to hide the page title disappears when importing Google Maps in the editor.
+* Fix: Slider - Resolved an slides per view ( pro option ) not working in editor.
```
Comment (adi3890) [inline README.md:224]:
```suggestion
* Fix: Slider - Resolved an issue where slides per view ( pro feature ) was not working in editor.
```

**Review by rajmohanreddy774 — APPROVED** [review]

**Review by AbhijitNage123 — APPROVED** [review]

**Review by adi3890 — APPROVED** [review]

**Pull request merged**